package.json

```diff
@@ -14,7 +14,7 @@
   ],
   "scripts": {
     "prebuild": "rimraf dist ethers && patch-package",
-    "build": "npm run compile:contracts && npm run build:typechain && npm run compile:ts && copyfiles build/contracts/**/*.json -u 1 dist && copyfiles \"ethers/**/*.d.ts\" dist",
+    "build": "npm run compile:contracts && npm run build:typechain && npm run compile:ts && copyfiles build/contracts/**/*.json -u 1 dist && copyfiles \"ethers/**/*.d.ts\" dist && shx cp -r contracts/* dist/contracts",
```

Comment:
Can we use `copyfiles` instead? As we already have that installed

Reply:
`copyfiles` equivalent looked weird to me `copyfiles contracts/**/**/**/**/*`.

Let me check if `copyfiles` has a better way of copying the whole folder without adding `**/` for each sub-folder.

Reply:
Actually, just adding quotes fixed the issue with `copyfiles`. Weird but this works now. Thanks

```sh
copyfiles \"contracts/**/*\" -u 1 dist/contracts
```